src/components/views/messages/TileErrorBoundary.tsx

```diff
@@ -83,16 +83,19 @@ export default class TileErrorBoundary extends React.Component<IProps, IState> {
 
             let viewSourceButton;
             if (mxEvent && SettingsStore.getValue("developerMode")) {
-                viewSourceButton = <AccessibleButton onClick={this.onViewSource} kind="link">
-                    { _t("View Source") }
-                </AccessibleButton>;
+                viewSourceButton = <>
+                    &nbsp;
+                    <AccessibleButton onClick={this.onViewSource} kind="link">
+                        { _t("View Source") }
+                    </AccessibleButton>
+                </>;
             }
 
             return (<li className={classNames(classes)} data-layout={this.props.layout}>
                 <div className="mx_EventTile_line">
                     <span>
                         { _t("Can't load this message") }
-                        { mxEvent && ` (${mxEvent.getType()})` }
+                        { mxEvent && ` (${mxEvent.getType()})` } &nbsp;
```

Comment:
```suggestion
                        { _t("Can't load this message") } &nbsp;
                        { mxEvent && `(${mxEvent.getType()})` } 
```

could there also be `&nbsp` in the if statement for `submitLogsButton`, so that we only add it if necessary?